spec/features/school_member/listing_school_members_spec.rb

```diff
@@ -0,0 +1,137 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe 'Listing school members', type: :request do
+  let(:headers) { { Authorization: UserProfileMock::TOKEN } }
+  let(:school) { create(:school) }
+  let(:owner) { create(:owner, school:) }
+  let(:teacher) { create(:teacher, school:) }
+  let(:students) { create_list(:student, 3, school:) }
+
+  before do
+    authenticated_in_hydra_as(owner)
+
+    student_attributes = students.map do |student|
+      { id: student.id, name: student.name, username: student.username }
+    end
+    stub_profile_api_list_school_students(school:, student_attributes:)
+
+    stub_user_info_api_for(teacher)
+  end
+
+  it 'responds 200 OK' do
+    get("/api/schools/#{school.id}/members", headers:)
+    expect(response).to have_http_status(:ok)
+  end
+
+  it 'responds with the class members JSON array' do
```

Comment:
think there's a couple of references to class members, will try and correct as I'm going

```suggestion
  it 'responds with the school members JSON array' do
```